command.go

```diff
@@ -88,8 +89,20 @@ func cmdTailscaleProxy(fs caddycmd.Flags) (int, error) {
 		return caddy.ExitCodeFailedStartup, fmt.Errorf("--to is required")
 	}
 
+	// strip "tailscale/" prefix if present
+	from, tsBind := strs.CutPrefix(from, "tailscale/")
+
 	// set up the downstream address; assume missing information from given parts
 	fromAddr, err := httpcaddyfile.ParseAddress(from)
+
+	var listen string
+	if tsBind {
+		listen = "tailscale/" + fromAddr.Host + ":" + fromAddr.Port
```

Comment:
Wonder if this will work OK with IPv6

Reply:
Seems like it?  I just tested with `caddy tailscale-proxy --from "tailscale/caddy:2000" --to :8000` and was able to access via tailscale ipv6 address.

Reply:
Ah, sorry, I meant `caddy tailscale-proxy --from <ip6_addr>` e.g. `--from 6240:3c33:...`

I suspect the IPv6 address would have to be surrounded in `[ ]`, which isn't bad (and is probably more correct) but users should just be aware if they do something like that.

Glad it's working over IPv6 though!

Reply:
nope... fails to parse if ipv6 address is specified.  I'll have to look into that